guides.md

```diff
@@ -32,13 +32,13 @@ This section will be dedicated to guides to learning Data Science, Machine Learn
 ### Existing guides
 
 #### Learning to learn
-  - [Virgilio's Teaching Strategy - Learning to Learn](https://github.com/virgili0/Virgilio/blob/master/Topics/teaching.md)
+  - [Deadlink] [Virgilio's Teaching Strategy - Learning to Learn](https://github.com/virgili0/Virgilio/blob/master/Topics/teaching.md)
```

Comment:
Hey @ShambhaviCodes great you pointed out its a deadlink but would even more fabulous if we could find working links and replace these non-working ones.

We have access to both the github repo and their website for this - I'm happy to help here as it's my project as well